### PR TITLE
Improving the docs.

### PR DESCRIPTION
**Pull request by haardikdharma10**

### CODE REVIEW

File: docs/K8sDeconstructedBasicOverview.org [hunk 1]
```diff
@@ -7,13 +7,13 @@ This is a collection of notes gathered from watching [[https://vimeo.com/2457781
   - All things about single application, runtime, disk
   - Should only be ONE application per service
 * Pod is smallest resource grouping in K8
```
Comment (RobertKielty) [inline docs/K8sDeconstructedBasicOverview.org:9]: Change to K8 to kubernetes
If we want to use the abbreviation it's k8s

File: docs/K8sDeconstructedBasicOverview.org [hunk 2]
```diff
@@ -76,15 +76,15 @@ This is a collection of notes gathered from watching [[https://vimeo.com/2457781
     - This sits above clusterIP
     - A way to traverse from outside the cluster into the nodes, I have a service ip, but each node gets a port assigned, now if outside the cluster that ip gets hit iptable rules (kube-proxy) will forward requests to that port to the right nodes
 *** ClusterIP
-    - core service for k8's now everything in the cluster knows the ip for the service. In service you can reference that ip by its lable, ie cache service once defined in services can be referenced by its name not ip, k8 will translate name into ip
+    - core service for k8's now everything in the cluster knows the ip for the service. In service you can reference that ip by its label, ie cache service once defined in services can be referenced by its name not ip, k8 will translate name into ip
```
Comment (RobertKielty) [inline docs/K8sDeconstructedBasicOverview.org:79]: k8's should be k8s

File: docs/K8sDeconstructedBasicOverview.org [hunk 3]
```diff
@@ -76,15 +76,15 @@ This is a collection of notes gathered from watching [[https://vimeo.com/2457781
...
 
 * k8 cloud Admin
```
Comment (RobertKielty) [inline docs/K8sDeconstructedBasicOverview.org:81]: Again k8 should be k8s

**Comment by haardikdharma10** [comment]
Done. Please have a cross-check.

**Comment by haardikdharma10** [comment]
@RobertKielty, please have a look.

**Comment by hh** [comment]
Thank for the doc updates @haardikdharma10 

Are you using APISnoop to to use the AuditSink in a live cluster or to query coverage from the latest release blocking PR jobs?

**Comment by haardikdharma10** [comment]
@hh, I used APISnoop to query the coverage from the latest release.

**Pull request merged**